vars/perfRegressionBareMetalSinglePipeline.groovy

```diff
@@ -0,0 +1,329 @@
+#!groovy
```

Comment:
What does the Single mean here ?

Reply:
The previous pipeline was called Matrix, so this is the opposite of that.

Reply:
You can just remove that word

Reply:
Done